### PR TITLE
chores : using latest popper dependency

### PR DESCRIPTION
**Pull request by vasikarla**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "create-react-context": "^0.3.0",
-    "popper.js": "^1.14.4",
+    "popper.js": "1.15.0",
```
Comment (TrySound) [inline package.json]: 1.15.0 is considered by `^1.14.4` range. If you want to upgrade transitive dependency remove and install again react-popper.

Comment (vasikarla) [reply package.json]: @TrySound : how do i handle this if im using popper dependency in my codebase?

Comment (TrySound) [reply package.json]:
```
yarn remove react-popper && yarn add react-popper
```
or
```
npm uninstall react-popper && npm install react-popper
```

**Comment by FezVrasta** [comment]
Closing as per the comments above

File: package.json (hunk 1 above)
Comment (vasikarla) [reply package.json]: let me try that!

**Pull request closed without merging**